sdk/security/keyvault/azcertificates/example_test.go

```diff
@@ -94,3 +96,59 @@ func ExampleClient_DeleteCertificate() {
 	// In a soft-delete enabled vault, deleted resources can be recovered until they're purged (permanently deleted).
 	fmt.Printf("Certificate will be purged at %v", *resp.ScheduledPurgeDate)
 }
+
+// This example uses `ImportCertificate` to import an existing PFX certificate.
+func ExampleClient_ImportCertificate_pfx() {
+	// Getting certificate data from locally stored PFX file then encoding to base64
+	data, err := os.ReadFile("PFX_CERT_PATH")
+	if err != nil {
+		// TODO: handle error
+	}
+	encodedData := base64.StdEncoding.EncodeToString(data)
+
+	// Assuming you already have a PFX containing your key pair, you can import it into Key Vault.
+	// You can do this without setting a policy, but the policy is needed if you want the private key to be exportable
+	// or to configure actions when a certificate is close to expiration.
```

Comment:
🤔 the comment seems to raise more questions because the example doesn't show doing those things. I'd remove the comment instead of showing the advanced API, to keep the example to the point